docs/build.md

```diff
@@ -68,7 +68,7 @@ git clone https://github.com/unicode-org/cldr.git
 ```
 2.  In the root folder of the `unicodetools` local working copy, create the `Generated/BIN` folder structure
     1. (Eclipse users can do this graphically by following the corresponding step in the Eclipse section below)
-    1. At the command-line: `cd unicodetools; mkdir -p output/Generated/BIN`
+    1. At the command-line: `cd unicodetools; mkdir -p Generated/BIN`
```

Comment:
I think it might be better if code that uses this structure created the directory automatically.